### PR TITLE
FF105 RelNote: TextEncoderStream, TextDecoderStream

### PR DESCRIPTION
**Pull request by hamishwillee**
FF105 adds support for [TextDecoderStream](https://developer.mozilla.org/en-US/docs/Web/API/TextDecoderStream)   - [TextEncoderStream](https://developer.mozilla.org/en-US/docs/Web/API/TextEncoderStream) in https://bugzilla.mozilla.org/show_bug.cgi?id=1486949

This is just the release note.

Other work can be tracked in #20572

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/105/index.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ This article provides information about the changes in Firefox 105 that will aff
 
 #### DOM
 
+- The [TextDecoderStream](/en-US/docs/Web/API/TextDecoderStream) and [TextEncoderStream](/en-US/docs/Web/API/TextEncoderStream) interfaces [Encoding API](/en-US/docs/Web/API/Encoding_API) are now supported ({{bug(1486949)}}).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/105/index.md:43]:
```suggestion
- The [Encoding API](/en-US/docs/Web/API/Encoding_API) now supports the [TextDecoderStream](/en-US/docs/Web/API/TextDecoderStream) and [TextEncoderStream](/en-US/docs/Web/API/TextEncoderStream) interfaces ({{bug(1486949)}}).
```

**Review by bsmth — APPROVED** [review]
Thanks, Hamish 👍🏻

File: files/en-us/mozilla/firefox/releases/105/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/105/index.md]: @bsmth The order I had this written in was deliberate.

What it said was that Firefox now supports these two APIs that are part of the Encoding API.

Now it says that the Encoding API supports these two interfaces. That is not the same thing. The encoding API always supported these two interfaces. It is just that Firefox has now implemented them.

Perhaps it is implied.

Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/105/index.md]: OK thanks, I had a feeling that was the case, sorry for the hasty merge. Minor improvement your way: https://github.com/mdn/content/pull/20640

Comment (hamishwillee) [reply files/en-us/mozilla/firefox/releases/105/index.md]: All good. Thanks.

**Pull request merged**